Frameworks/UIKit/UITextField.mm

```diff
@@ -307,15 +330,15 @@ - (void)setTextColor:(UIColor*)color {
  @Status Interoperable
 */
 - (UIColor*)textColor {
-    WUXMSolidColorBrush* colorBrush = nil;
+    Media::SolidColorBrush colorBrush = nullptr;
     if (_secureTextMode) {
-        colorBrush = rt_dynamic_cast<WUXMSolidColorBrush>(_passwordBox.foreground);
+        colorBrush = _passwordBox.Foreground().try_as<Media::SolidColorBrush>();
```

Comment:
Maybe. Using try_as just preserves the existing logic.

---
In reply to: [109244995](https://github.com/Microsoft/WinObjC/pull/2382#discussion_r109244995) [](ancestors = 109244995)